nearup

```diff
@@ -19,7 +21,11 @@ logging.basicConfig(
     '%(asctime)s.%(msecs)03d %(levelname)s %(module)s - %(funcName)s: %(message)s',
     datefmt='%Y-%m-%d %H:%M:%S',
     handlers=[
-        logging.FileHandler(os.path.expanduser('~/.nearup/logs/nearup.log')),
+        handlers.RotatingFileHandler(
+            os.path.expanduser('~/.nearup/logs/nearup.log'),
+            maxBytes=1024 * 1024,
+            backupCount=10,
```

Comment:
what does `backupCount` mean in this context?

Reply:
for logs to be log.1 log.2 ... log.10

Reply:
Does that mean we could only restart 10 times?

Reply:
No after that starts circularly I believe